openreview/conference/templates/programchairWebfield.js

```diff
@@ -26,6 +26,9 @@ var reviewerSummaryMap = {};
 var allReviewers = [];
 var conferenceStatusData = {};
 
+var PC_PAPER_ASSIGNMENT = true;
```

Comment:
instead of using a boolean, I would show the tags if the invitation exits.

Reply:
This change has been made.